### ISSUE TITLE
[Bug]: Accessibility - In multi-select drop down, NVDA, cannot navigate selected items using left or right arrow keys

### ISSUE THREAD

**Issue by binhphan009** [microsoft/fluentui#29094]
### Library

React Northstar / v0 (@fluentui/react-northstar)

### System Info

```shell
System:
    OS: Windows 10 10.0.19044
    CPU: (16) x64 11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz
    Memory: 2.77 GB / 15.73 GB
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (116.0.1938.69)
    Internet Explorer: 11.0.19041.1566
```

### Are you reporting Accessibility issue?

Yes

### Reproduction

https://codesandbox.io/s/dropdown-multi-select-ljzjlz

## Actual Behavior
Repro Steps:
1. Open NVDA
2. Focus on to Multi select
3. Select 2 items in the DropDown

<img width="358" alt="image" src="https://github.com/microsoft/fluentui/assets/105278576/57a616d9-ad8e-45ef-88b2-5b3591d5fae1">

5. Cannot use left and right arrow keys to navigate selected items.
<img width="481" alt="image" src="https://github.com/microsoft/fluentui/assets/105278576/7e5d9db5-a8b9-4994-a0ce-aba04bc5cbe6">

![arrow key](https://github.com/microsoft/fluentui/assets/105278576/3281b8ef-49db-4393-8f9a-0ee617b51fb5)

## Expected Behavior
You can use the left and right arrow keys to navigate through selected items.

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by smhigley** [comment]
The selected items in the multiselect dropdown component are marked up as options within a listbox. NVDA does not allow virtual cursor navigation inside a listbox -- something you can verify in this basic ARIA Authoring Practices example: https://www.w3.org/WAI/ARIA/apg/patterns/listbox/examples/listbox-scrollable/

This isn't a bug, it's just how NVDA behaves -- the options are navigable in forms mode.
